### PR TITLE
Now shows when a registration have been clicked, closes #9

### LINKED ISSUE #9: When clicking a registration, it should be marked as copy in the UI
This raises the following questions:
- Should it toggle back, if pressed again?
- Should toggling back copy the text again? 
- - if not - should a toast say that the item is no longer "clicked"?

### PR DESCRIPTION
**Pull request by Julius-Bendt**
Messed up branching, merge https://github.com/Julius-Bendt/tid.jub.dk/pull/19 first
closes #9

### CODE REVIEW

File: src/HomeView.vue [hunk 1]
```diff
@@ -140,7 +146,16 @@ function setOrAddRegistration(input: IRegistration) {
...
+    clicked: clicked
   } as IRegistration)
 }
+
+function registrationClicked(input: IRegistration) {
+  const registration: IRegistration = formattedRegistrations.value.get(
+    input.letter
+  ) as IRegistration
+
+  registration.clicked = !registration.clicked // I did not except this to work without modifying the map it self. apparently it does indeed update.
```
Comment (Neophear) [inline src/HomeView.vue:159]: expect?

File: src/components/RegistrationTable.vue [hunk 2]
```diff
@@ -12,8 +12,9 @@
       <tr
         v-for="registration in props.registrations"
         :key="registration.letter"
-        @click="copyToClipboard(registration.description)"
+        @click="clickRegistration(registration)"
         class="odd:bg-gray-900 even:bg-gray-800 border-b border-gray-700 hover:bg-primary text-text cursor-pointer transition-colors"
+        :class="{ 'line-through': registration.clicked }"
```
Comment (Neophear) [inline src/components/RegistrationTable.vue:17]: Wouldnt some kind of dimmed down color be more visually pleasing?

File: src/components/RegistrationTable.vue [hunk 3]
```diff
@@ -28,18 +29,18 @@
...
+const emit = defineEmits(['registrationClicked'])
+
 const props = defineProps({
   registrations: { type: Array<IRegistration>, default: () => [] },
   bar: Number
 })
 
 function getTimePeriods(registration: IRegistration): string {
-  const ranges = []
+  const ranges: string[] = []
```
Comment (Neophear) [inline src/components/RegistrationTable.vue:43]: lol

**Review by Neophear — CHANGES_REQUESTED** [review]

File: src/HomeView.vue (hunk 1 above)
Comment (Julius-Bendt) [reply src/HomeView.vue]: Fixed

File: src/components/RegistrationTable.vue (hunk 3 above)
Comment (Julius-Bendt) [reply src/components/RegistrationTable.vue:46]: Got a warning 🤷‍♂️

**Review by Neophear — APPROVED** [review]

**Pull request merged**